backend/tests/engine/projects/conftest.py

```diff
@@ -77,3 +81,76 @@ def data_for_quadratic_funding(projects):
     matched_rewards = 35000
 
     return allocations, matched_rewards
+
+
+@pytest.fixture
+def dataset_1_for_capped_quadratic_funding(projects):
+    matched_rewards = 35000
+    capped_matched_rewards = matched_rewards * MR_FUNDING_CAP_PERCENT
+    allocations = [
+        ProjectSumAllocationsDTO(projects[0], 1000),
+        ProjectSumAllocationsDTO(projects[1], 2000),
+        ProjectSumAllocationsDTO(projects[2], 3000),
+        ProjectSumAllocationsDTO(projects[3], 4000),
+        ProjectSumAllocationsDTO(projects[4], 5000),
+        ProjectSumAllocationsDTO(projects[5], 6000),
+        ProjectSumAllocationsDTO(projects[6], 7000),
+        ProjectSumAllocationsDTO(projects[7], 8000),
+        ProjectSumAllocationsDTO(projects[8], 9000),
+        ProjectSumAllocationsDTO(projects[9], 10000),
+    ]
+
+    expected_distribution = {
```

Comment:
It seems incorrect to me, `matched_rewards` is 35000, but sum of capped matched rewards is 55000

Reply:
changed